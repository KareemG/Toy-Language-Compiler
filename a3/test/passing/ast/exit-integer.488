% Exit with integer statement

{
    var x : integer

    x := 0

    while x < 10 do
    {
        exit
    }
}