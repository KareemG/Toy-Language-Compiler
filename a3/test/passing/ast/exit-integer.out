 { 
 var x : integer
 x : = 0
 while (x) < (10) do
 { 
 exit
 } 
 }
